assets/wizards/site-design/views/main/index.js

```diff
@@ -0,0 +1,275 @@
+	SelectControl,
+	ToggleControl,
+	SectionHeader,
+	ImageUpload,
+	hooks,
+	Button,
+	Grid,
+	WebPreview,
+} from '../../../../components/src';
+import ThemeSelection from '../../components/theme-selection';
+import { getFontsList, getFontImportURL, LOGO_SIZE_OPTIONS, parseLogoSize } from './utils';
+import './style.scss';
+
+const Main = ( {
+	wizardApiFetch,
+	setError,
+	renderPrimaryButton,
+	buttonText,
+	hasPreview = true,
+} ) => {
+	const [ themeSlug, updateThemeSlug ] = useState();
+	const [ mods, updateMods ] = hooks.useObjectState();
+
+	const updateSettings = response => {
+		updateMods( response.theme_mods );
+		updateThemeSlug( response.theme );
+	};
+	useEffect( () => {
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/newspack-setup-wizard/theme',
+		} )
+			.then( updateSettings )
+			.catch( setError );
+	}, [] );
+
+	const saveSettings = () =>
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/newspack-setup-wizard/theme/',
+			method: 'POST',
+			data: { theme_mods: mods, theme: themeSlug },
+			quiet: true,
+		} )
+			.then( updateSettings )
+			.catch( setError );
+
+	return (
+		<>
+			<SectionHeader
+				title={ __( 'Theme', 'newspack' ) }
+				description={ __( 'Activate a theme you like to get started', 'newspack' ) }
+			/>
+			<ThemeSelection theme={ themeSlug } updateTheme={ updateThemeSlug } />
+			<SectionHeader
+				title={ __( 'Colors', 'newspack' ) }
+				description={ __( 'Define your primary and secondary colors', 'newspack' ) }
+			/>
+			<Grid gutter={ 32 }>
+				<ColorPicker
+					label={ __( 'Primary' ) }
+					color={ mods.primary_color_hex }
+					onChange={ updateMods( 'primary_color_hex' ) }
+				/>
+				<ColorPicker
+					label={ __( 'Secondary' ) }
+					color={ mods.secondary_color_hex }
+					onChange={ updateMods( 'secondary_color_hex' ) }
+				/>
+			</Grid>
+			<SectionHeader
+				title={ __( 'Typography', 'newspack' ) }
+				description={ __( 'Pick the font pairing to use throughout your site', 'newspack' ) }
+			/>
+			<Grid gutter={ 32 }>
+				<SelectControl
+					label={ __( 'Headings', 'newspack' ) }
+					optgroups={ getFontsList( true ) }
+					value={ mods.font_header }
+					onChange={ ( value, group ) =>
+						updateMods( {
+							font_header: value,
+							custom_font_import_code: getFontImportURL( value ),
+							font_header_stack: group?.fallback,
+						} )
+					}
+				/>
+				<SelectControl
+					label={ __( 'Body', 'newspack' ) }
+					optgroups={ getFontsList() }
+					value={ mods.font_body }
+					onChange={ ( value, group ) =>
+						updateMods( {
+							font_body: value,
+							custom_font_import_code_alternate: getFontImportURL( value ),
+							font_body_stack: group?.fallback,
+						} )
+					}
+				/>
+			</Grid>
+			<ToggleControl
+				checked={ mods.accent_allcaps === true }
+				onChange={ updateMods( 'accent_allcaps' ) }
+				label={ __( 'Use all-caps for accent text', 'newspack' ) }
+			/>
+			<SectionHeader
+				title={ __( 'Header', 'newspack' ) }
```

Comment:
(Not a blocker) I know that the logo image preview is a preview of the logo only, but its place in the UI makes it look like it represents a preview of the header itself (especially because it updates with different color settings). Would be great if changing other options updated the preview as well. Something like:

* Alignment: We can update the `background-position` of the image upload component to left/center/right.
* Size (Header): We could update the dimensions of the image upload component itself.
* Size (Logo): We could update the dimensions of the image inside the image upload component to approximate the rendered size.
* Display Site Title/Tagline: These might be tricker to render with respect to the logo, but I really wanted to see them if we can figure out a way to show them easily on top of the image logo preview.

The above is definitely more of a nice-to-have and not a blocker.

Reply:
Hm, that would duplicate the preview feature. I'd rather opt for a single complete preview than fragmented previews. WDYT @thomasguillot?

Reply:
> that would duplicate the preview feature. I'd rather opt for a single complete preview than fragmented previews

Agreed.

> Display Site Title/Tagline

Speaking of the Site Title/Tagline. Can we remove the toggles? (cc @adekbadek) If there's a logo then Site Title and Tagline should be set to "hidden". If a publication wants to display the Site Title and/or the Tagline alongside their logo, then they can always use the Customizer (fwiw it only represents ~8% of all our current publishers)

Reply:
Done in a5b9b9020db7a1c98690e2b32c03feae3234d9f0